.github/scripts/build_daprbundle.py

```diff
@@ -120,12 +120,16 @@ def downloadBinaries(dir):
     unpack_archive(cli_filepath,dir)
     os.remove(cli_filepath)
 
+def getFileName(image):
```

Comment:
add doc strings on what this message expects as input and what it outputs ,,, 

In general that can be done for any function ...

Reply:
Done!